runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/Credential/EditCredential/CredentialAttribute/CredentialAttributeDetails.tsx

```diff
@@ -77,7 +76,7 @@ const CredentialAttributeDetails: FC<CredentialAttributeDetailsProps> = ({ handl
                                         onMouseDown={handleMouseDownValue}
                                         onMouseUp={handleMouseUpValue}
                                     >
-                                        {showValue ? <VisibilityOff /> : <Visibility />}
+                                        {maskValue ? <Visibility /> : <VisibilityOff />}
```

Comment:
if you're doing another changes it's good to rename Visibility to VisibilityOn

Reply:
but without context it's hard to understand so I suggest changin components names to \<RevealedInput />, \<MaskedInput />

Reply:
these are not my components, these are the MUI ones :)

Reply:
I just changes the orther to make it more clear